demo/fiber/oneshot_test/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,264 @@
+package org.example;
+
+/*
+ * Copyright (c) 2018, 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+//import jdk.internal.vm.Continuation;
+//import jdk.internal.vm.ContinuationScope;
+
+import java.util.concurrent.TimeUnit;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import org.openjdk.jmh.runner.*;
+
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+@State(Scope.Thread)
```

Comment:
Just a question:
Which scope, `Scope.Benchmark` or `Scope.Thread`, should be used here?

Reply:
I agree with the code here, i.e. we should use Scope.Thread. 
Considering that Main.java has two inner class, which simulate two different scenarios, and has different benchmarks, I guess the purpose of this file is to measure the performance of Continuation in different scenarios, such as yielding before or after a single run. Therefore, we need to make sure each thread has its own parameters, avoiding contention and interference among threads

Reply:
> Therefore, we need to make sure each thread has its own parameters, avoiding contention and interference among threads

If I understand that correctly, the parameters, exactly `paramCount` and `stackDepth`, are read-only in a single benchmark run.

Reply:
Sorry I didn't notice that at first, then I think Scope.Benchmark would be OK